### DISCUSSION TITLE
Version 1.16.5

### DISCUSSION THREAD

**Discussion by DiumStream-tech** [Sinytra/Connector#1495] [Ideas]
Hi 

I would like to know if it is possible for you to release a version 1.16.5 for our modpack https://www.curseforge.com/minecraft/modpacks/pack-sio

Because we use sodium even a fork for forge which is full bugs is made of crap all the players we have already reported the bugs or creator of the fork but we do nothing 

It would be a great blessing to have your mods on our modpack 

Sincerely: DiumStream / Zipname

**Comment by coco875** [comment]
No backport are planned.

**Comment by DiumStream-tech** [comment]
?
